src/os/ui/featureedit.js

```diff
@@ -980,8 +1010,20 @@ os.ui.FeatureEditCtrl.prototype.loadFromFeature_ = function(feature) {
       if (coordinate) {
         this['pointGeometry'] = {
           'lon': coordinate[0],
-          'lat': coordinate[1]
+          'lat': coordinate[1],
+          'alt': coordinate[2]
         };
+
+        var altU = this['options']['feature']['values_']['ALTITUDE_UNITS'];
+
+        // else if runs when the page is refreshed
+        if (coordinate[2] !== null && coordinate[2] !== undefined && altU !== null && altU !== undefined) {
```

Comment:
`val != null` is functionally equivalent to `val !== null && val !== undefined`. The former uses loose equality, and `null == undefined` in JavaScript.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness